import-scripts/clone_db_wrapper.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Script for running pre-import steps
+# Consists of the following:
+# - Determine which database is "production" vs "not production"
+# - Drop tables in the non-production database
+# - Clone the production database into the non-production database
+
+PORTAL_SCRIPTS_DIRECTORY=$1
+if [ -z $PORTAL_SCRIPTS_DIRECTORY ]; then
+    PORTAL_SCRIPTS_DIRECTORY="/data/portal-cron/scripts"
+fi
+if [ ! -f $PORTAL_SCRIPTS_DIRECTORY/automation-environment.sh ] ; then
+    echo "`date`: Unable to locate automation_env, exiting..."
```

Comment:
probably better to spell out the entire filename which is unfound : 'automation_environment.sh'

Reply:
done